app/views/main/header.php

```diff
@@ -14,5 +14,15 @@
     <link rel="stylesheet" href="/darkmode_style.css">
   <?php endif; ?>
 
+  <?php if($data["config"]["background_image"] != ""): ?>
```

Comment:
Let's add isset() to the if statement, something more like: `if(isset($data["config"]["background_image"]) && $data["config"]["background_image"] != "")`

(On line 17 of app/views/main/header.php)

Reason:  If someone updates to this commit from a previous commit, they will get the following error:

> Notice: Undefined index: background_image in /var/www/Intranet-Home-Page/app/views/main/header.php on line 17

By adding the `isset()` check **before** actually accessing $data["config"]["background_image"] index, we can avoid throwing an error when encountering an older version of the config.

If we can fix that, this looks great and I'd say it's good to go!

Thank you for your interest in the project and your contribution to the open-source community!

Reply:
Excellent catch! It has been added. I look forward to adding some more ideas I have to this project!